.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+npm install
+npm run linter
+npm run test-unit
+npm run test-integration
```

Comment:
maybe pre-push is more appropriate here? Run this through, if you have failing tests or failing linting, you cannot commit

Reply:
Yes I agree, moved to pre-push